src/main/resources/application.properties

```diff
@@ -6,8 +6,11 @@ spring.sql.init.mode=always
 spring.sql.init.platform=postgres
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 
+spring.datasource.username=postgres
+spring.datasource.password=postgres
+
 
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.generate-ddl=true
 
-server.port=8081
+springdoc.swagger-ui.path=/swagger-ui
```

Comment:
I recommend `/api/v1/swagger-ui`.

Reply:
It is already `api/v1/overworld/swagger-ui/` because I added `server.servlet.context-path=/api/v1/overworld` in `application.properties`

Reply:
That is the application.properties, where is that key?